OpenRA.Game/Settings.cs

```diff
@@ -221,7 +222,6 @@ public class GameSettings
 		public string AuthProfile = "player.oraid";
 
 		public bool AllowZoom = true;
-		public Modifiers ZoomModifier = Modifiers.Ctrl;
```

Comment:
I originally pushed for this modifier because the previous solution wasn't a "real" zoom, so shouldn't have owned the mousewheel by default. This is no longer true, so IMO we can benefit more by dropping the option. This PR is promoting zooming to a top-tier feature, and the changes to the zoom reset hotkey mean that scrolling is now the only way to access the additional zoom levels in spectator/editor modes.

Reply:
In that case I think we should remove the `AllowZoom` checkbox as well then.

Regarding accessing the additional zoomlevels without the mouse wheel: We could use the '+' and '-' keys for that.

Reply:
As per our discussion on Discord, could this be optional?